spring-data-sample/README.md

```diff
@@ -0,0 +1,269 @@
+      "customer_id": "INT",
+      "name": "TEXT",
+      "credit_limit": "INT",
+      "credit_total": "INT"
+    }
+  },
+  "sample.orders": {
+    "transaction": true,
+    "partition-key": [
+      "customer_id"
+    ],
+    "clustering-key": [
+      "timestamp"
+    ],
+    "secondary-index": [
+      "order_id"
+    ],
+    "columns": {
+      "order_id": "TEXT",
+      "customer_id": "INT",
+      "timestamp": "BIGINT"
+    }
+  },
+  "sample.statements": {
+    "transaction": true,
+    "partition-key": [
+      "order_id"
+    ],
+    "clustering-key": [
+      "item_id"
+    ],
+    "columns": {
+      "order_id": "TEXT",
+      "item_id": "INT",
+      "count": "INT"
+    }
+  },
+  "sample.items": {
+    "transaction": true,
+    "partition-key": [
+      "item_id"
+    ],
+    "columns": {
+      "item_id": "INT",
+      "name": "TEXT",
+      "price": "INT"
+    }
+  }
+}
+```
+
+All the tables are created in the `sample` namespace.
+
+- `sample.customers`: a table that manages customers' information
+- `credit_limit`: the maximum amount of money a lender will allow each customer to spend when using a credit card
+- `credit_total`: the amount of money that each customer has already spent by using the credit card
+- `sample.orders`: a table that manages order information
+- `sample.statements`: a table that manages order statement information
+- `sample.items`: a table that manages information of items to be ordered
+
+The Entity Relationship Diagram for the schema is as follows:
+
+![ERD](images/ERD.png)
+
+### Transactions
+
+The following five transactions are implemented in this sample application:
+
+1. Getting customer information
+2. Placing an order by credit card (checks if the cost of the order is below the credit limit, then records order history and updates the `credit_total` if the check passes)
+3. Getting order information by order ID
+4. Getting order information by customer ID
+5. Repayment (reduces the amount in the `credit_total`)
+
+### Configuration
+
+Configurations for the sample Spring Boot application are as follows:
+
+```application.properties
+spring.datasource.driver-class-name=com.scalar.db.sql.jdbc.SqlJdbcDriver
+spring.datasource.url=jdbc:scalardb:
+?scalar.db.storage=cassandra\
+&scalar.db.contact_points=localhost\
+&scalar.db.username=cassandra\
+&scalar.db.password=cassandra
+```
+
+Since this sample application uses Cassandra, as shown above, you need to configure your settings for Cassandra in this configuration.
+
+## Setup
+
+To start Cassandra and load the schema, you need to run the following `docker-compose` command:
+
+```shell
+$ docker-compose up -d
+```
+
+Please note that starting the containers may take more than one minute.
+
+### Initial data
+
+After the containers have loaded, you need to load the initial data by running the following command:
+
+```shell
+$ ./gradlew run --args="LoadInitialData"
+```
+
+After the initial data has loaded, you will see the following:
```

Comment:
```
- And the following data will be loaded:
+ After the initial data has loaded, you will see the following:
```

@josh-wong Noticed that users need to use Cassandra CLI or ScalarDB CLI tool to see the stored records and I think it's a bit hard. How about just describing the data has been loaded like as follows?

```
After the initial data has loaded, the following records should be stored in the tables:
```

Reply:
@komamitsu Thank you for clarifying. LGTM!